File: backend/LoanComparer.Application/DTO/UserDTO/UserForRegistrationDTO.cs
namespace LoanComparer.Application.DTO.UserDTO
{
    public record UserForRegistrationDTO(
        string FirstName,
        string LastName,
        string Email,
        JobTypeDTO JobType,
        int IncomeLevel,
        GovernmentIdDTO GovernmentId,
        string Password,
        string ConfirmPassword,
        string ClientURI);
}

File: frontend/src/app/authentication/models/user-for-registration-dto.ts
import { JobTypeDTO } from '../../core/models/job-type-dto';
import { GovernmentIdDTO } from '../../core/models/government-id-dto';

export interface UserForRegistrationDTO {
  firstName: string;
  lastName: string;
  email: string;
  jobType: JobTypeDTO;
  incomeLevel: number;
  governmentId: GovernmentIdDTO;
  password: string;
  confirmPassword: string;
  clientURI: string;
}
